.github/workflows/build.yml

```
@@ -0,0 +1,41 @@
+    branches:
+      - Category-entity  # Your feature branch
+  pull_request:
+    branches:
+      - main           # Base branch for the PR
+
+jobs:
+  build-and-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)